generated/.gitignore

```diff
@@ -0,0 +1,64 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Do we actually need this file under source control? This file is generated.

Reply:
removed in 09b5f9cd6b08f41f9226ab61decbf54bb237b99b